src/lib/math.ts

```diff
@@ -4,17 +4,23 @@ import type { BN } from './bignumber/bignumber'
 import { bn, bnOrZero } from './bignumber/bignumber'
 
 // Converts from base unit to a precision/ish number
-// If no displayDecimals are provided, it will use the full precision of the number being converted
-// If displayDecimals are provided, it will use that precision, to return e.g a human, precision, or number stripped to any arbitrary decimal places
+// - If no displayDecimals are provided, it will use the full precision of the number being converted
+// - If displayDecimals are provided, it will use that precision, to return e.g a human, precision, or number stripped to any arbitrary decimal places
+// and use the ROUND_DOWN rounding mode
 export const fromBaseUnit = (
   value: BigNumber.Value,
   precision: number,
   displayDecimals?: number,
 ): string => {
-  return bnOrZero(value)
-    .div(bn(10).pow(precision))
-    .decimalPlaces(displayDecimals ?? precision, BigNumber.ROUND_DOWN)
-    .toFixed()
+  const precisionNumber = bnOrZero(value).div(bn(10).pow(precision))
+
+  if (typeof displayDecimals === 'number') {
```

Comment:
`.decimalPlaces(0)` is perfectly valid, so we can't use `if (displayDecimals)` to check for the argument being passed, since it would evaluate to false in case of 0